### PR TITLE
Add PR scanner to hammer-cli-csv

### PR DESCRIPTION
**Pull request by akofink**
Automatically link PRs to redmine issues when submitted on GitHub

@thomasmckay

### CODE REVIEW

File: config/repos.yaml [hunk 1]
```diff
@@ -176,6 +176,7 @@ Katello/foreman-gutterball:
 Katello/foreman_sam:
   redmine: katello
 Katello/hammer-cli-csv:
+  pr_scanner: true
```
Comment (domcleal) [inline config/repos.yaml]: The project doesn't seem to use Jenkins for pull requests, are you sure you need this enabled?

Comment (akofink) [inline config/repos.yaml]: @domcleal I'm not sure what needs to be changed, but the hammer-cli-csv PRs are not getting linked properly to the redmine issue when submitted to GitHub. What do you think is missing/wrong with the config given that information?

Comment (domcleal) [inline config/repos.yaml]: Probably the webhook.

**Comment by akofink** [comment]
Closing as not needed (will add the web hook in hammer-cli-csv repo settings)

**Pull request closed without merging**